cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -162,6 +170,35 @@ class ExamplesFunTest : StringSpec() {
             report shouldHaveSize 1
         }
 
+        "emails can be send using notifications.kts" {
+            val greenMail = GreenMail()
+            greenMail.setUser("no-reply@ossreviewtoolkit.org", "no-reply@ossreviewtoolkit.org", "pwd")
+            greenMail.start()
+
+            val ortResult = File("src/funTest/assets/semver4j-analyzer-result.yml").readValue<OrtResult>()
+            val notifier = Notifier(
+                ortResult, NotifierConfiguration(
+                    EmailConfiguration(
+                        hostName = "localhost",
+                        port = 3025,
```

Comment:
It is preferable to use a dynamic port here if possible, rather than a hard-coded one, which might already be used in a specific environment. I think, the documentation of GreenMail mentions that this is supported.